include/Core/Metas/Modules/DeleteCommentMetaModule.php

```diff
@@ -154,4 +154,13 @@ protected function get_success_message( $items_deleted ) {
 		/* translators: 1 Number of posts deleted */
 		return _n( 'Deleted comment meta field from %d comment', 'Deleted comment meta field from %d comments', $items_deleted, 'bulk-delete' );
 	}
+
+	/**
+	 * Schedule job title.
+	 *
+	 * @return string humane readable title
+	 */
+	protected function get_cron_action_name(){
```

Comment:
Let's add a new item to the `$this->messages` array called 'cron_name' and use it from base class.

In the child classes just set the value of the `$this->messages` array in the `initialize` method. That way we can remove an extra method from each sub class.